css/filmstrip/_tile_view.scss

```diff
@@ -2,7 +2,9 @@
  * CSS styles that are specific to the filmstrip that shows the thumbnail tiles.
  */
 .tile-view {
-
+    video {
```

Comment:
I saw that now the face centering is enabled on stage view as well but the transition is not applied there.

Reply:
just the detection is enabled in order to allow receivers to center detected faces in case they're in tile view. There's no point in adding object-position when in stage view, since it already displays the whole video area

Reply:
For the large video I agree. But in the same time we display the filmstrip in stage view and I don't think we show the whole video in the thumbnails.

Also now the stage view filmstrip resize PR is merged, we technically can have a tile view filmstrip displayed when you are in stage view.

Anyway my point on the original comment was that when I'm testing and looking into the code we actually do set the object-position to the filmstrip videos in stage view and we don't have transition in the css. So I think we either need to stop setting the object-position for stage view in the `Thumbnail` component  or we need to add transition (which IMHO is the way to go).

Reply:
Good point, missed that. Moved transition declaration to apply to any video, so if object-position changes it will take effect.